### PR TITLE
chore: added helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     |✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                      |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-name: ts-server-boilerplate
-description: A Helm chart for ts-server-boilerplate service
+name: layer-spec
+description: "layer spec app deployement"
```
Comment (asafMasa) [inline helm/Chart.yaml]: Why do you need "" ?

File: helm/values.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ global:
   tracing: {}
   metrics: {}
   environment: {}
+  imagePullPolicy: Always
```
Comment (asafMasa) [inline helm/values.yaml]: check with yossi

File: helm/values.yaml [hunk 3]
```diff
@@ -15,8 +16,22 @@ cloudProvider:
   flavor: openshift
 
 image:
-  repository: ts-server-boilerplate
-  tag: 'latest'
+  repository: layer-spec-registry
+  tag: latest
```
Comment (asafMasa) [inline helm/values.yaml]: tag should have ''?

File: openapi3.yaml [hunk 4]
```diff
@@ -20,7 +20,6 @@ paths:
           required: true
           schema:
             type: string
-            format: uuid
```
Comment (asafMasa) [inline openapi3.yaml:23]: why?

**Review by asafMasa — CHANGES_REQUESTED** [review]
:+1:

File: openapi3.yaml (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply openapi3.yaml:23]: fixed in 9489fb6

File: helm/values.yaml (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml]: fixed in 9489fb6

File: helm/values.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml]: removed for now in 9489fb6

File: helm/Chart.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml]: fixed in 9489fb6

File: helm/templates/postgres-secrect.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  {{- if .Values.env.db.sslEnabled }}
+  caFile: {{ .Files.Get $caPath | quote }}
+  keyFile: {{ .Files.Get $keyPath | quote }}
+  certFile: {{ .Files.Get $certPath | quote }}
+  {{- else }}
+  caFile: ""
+  keyFile: ""
+  certFile: ""
+  {{- end }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/postgres-secrect.yaml:22]: update

**Review by asafMasa — APPROVED** [review]

File: helm/Chart.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-name: ts-server-boilerplate
-description: A Helm chart for ts-server-boilerplate service
+name: "layer-spec"
+description: "layer spec app deployement"
```
Comment (Yossizz) [inline helm/Chart.yaml]: layer spec chart

File: helm/templates/configmap.yaml [hunk 7]
```diff
@@ -20,4 +20,9 @@ data:
   TELEMETRY_METRICS_URL: {{ $metricsUrl }}
   {{ end }}
   LOG_LEVEL: {{ .Values.env.logLevel | quote }}
+  DB_HOST: {{ .Values.env.db.host | quote }}
+  DB_PORT: {{ .Values.env.db.port | quote }}
```
Comment (Yossizz) [inline helm/templates/configmap.yaml]: remove quote

File: helm/templates/ingress.yaml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {{- $releaseName := .Release.Name -}}
```
Comment (Yossizz) [inline helm/templates/ingress.yaml:1]: remove if no necccery

File: helm/templates/postgres-secrect.yaml (hunk 5 above)
Comment (Yossizz) [inline helm/templates/postgres-secrect.yaml]: add empty space

File: helm/templates/route.yaml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {{- $releaseName := .Release.Name -}}
```
Comment (Yossizz) [inline helm/templates/route.yaml:1]: remove

**Review by Yossizz — CHANGES_REQUESTED** [review]

File: helm/Chart.yaml (hunk 6 above)
Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml]: fixed

Comment (CL-SHLOMIKONCHA) [reply helm/Chart.yaml]: fixed

File: helm/templates/configmap.yaml (hunk 7 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/configmap.yaml]: removed

File: helm/templates/postgres-secrect.yaml (hunk 5 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/postgres-secrect.yaml]: fixed

File: helm/templates/ingress.yaml (hunk 8 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/ingress.yaml:1]: necessary

File: helm/templates/route.yaml (hunk 9 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/route.yaml:1]: necessary

**Review by Yossizz — APPROVED** [review]

**Pull request merged**